mips/exc.S

```diff
@@ -143,6 +204,74 @@ user_exc_leave:
         LOAD_REG(t0, PC, k0)
         mtc0    t0, C0_EPC   
 
+
+
+        # Load FPU context process makes use of it
+        # by defaul assume that process does not, and disable FPU
+        mfc0 t0, C0_STATUS
+        li t1, ~(SR_CU1)
+        and t0, t1
+        mtc0 t0, C0_STATUS
+
+        # Skip if process is not using FPU
+        LOAD_PCPU(t0)
```

Comment:
Please load `curthread` once into `k1` register in [exc.S:122](https://github.com/cahirwpz/mimiker/blob/master/mips/exc.S#L122) and then use it throughout following code.